js/factorize.js

```diff
@@ -0,0 +1,112 @@
+//var central_repository_server = "https://kinto.dev.mozaws.net/v1";
+//var bucket = 'central-repository';
+//var collection = 'users';
+var headers = authenticate();
+
+function getUserURL(user_storage_url)
+{
+  //takes the url of the storage server of the user.
+   storageServer - user_storage_url;
+  // if user_storage_url has a value, it return this value
+  // else it returns a default value:https://kinto.dev.mozaws.net/v1
+  if (storageServer == ''){
+    storageServer = "https://kinto.dev.mozaws.net/v1"  //default
+  }
+
+  //maybe we can validate the url entered here.
+   return storageServer; //i.e. url
+}
+
+function registerUserURL(user_id,url){
```

Comment:
From the code, sometime you will return ``undefined``, sometime you will return ``"record created"``